src/components/Map/Map.tsx

```diff
@@ -127,14 +149,16 @@ function Map() {
 
   const handleSearch = (e:any) => {
```

Comment:
We're getting what we expect when a user first enters a zipcode and a radius, what we'd really like is for the user to be able to navigate back to products or settings then back to the map and still be able to see the zip and radius map view that they searched.  Right now when the user revisits, the map view goes back to the default, the pins and url are holding state, just the map view has to be figured out.